<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Agent</class>
 <widget class="QDialog" name="Agent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>545</height>
   </rect>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>801</width>
     <height>551</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="styleSheet">
     <string notr="true">
background-color:#121111;


</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_prenom">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>260</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : transparent;
border:none;
color : #717072;
border-bottom : 1px solid  #717072;
font-size: 18px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>390</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;
color : grey;</string>
     </property>
     <property name="text">
      <string> LISTE AGENT</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Sportify-QT-project-master/worker.png</normaloff>../../Sportify-QT-project-master/worker.png</iconset>
     </property>
    </widget>
    <widget class="QLineEdit" name="recherche">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>151</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : transparent;
border:none;
color : #717072;
border-bottom : 1px solid  #717072;
font-size: 18px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rechercher">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>150</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;
color : grey;</string>
     </property>
     <property name="text">
      <string>Rechercher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>624</x>
       <y>390</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;
color : grey;</string>
     </property>
     <property name="text">
      <string>load data</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Sportify-QT-project-master/load.png</normaloff>../../Sportify-QT-project-master/load.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
        font-size:16px;</string>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cin">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>320</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : transparent;
border:none;
color : #717072;
border-bottom : 1px solid  #717072;
font-size: 18px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : transparent;
border:none;
color : #717072;
border-bottom : 1px solid  #717072;
font-size: 18px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_mdp">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>370</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : transparent;
border:none;
color : #717072;
border-bottom : 1px solid  #717072;
font-size: 18px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
        font-size:16px;</string>
     </property>
     <property name="text">
      <string>CIN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter_Ag">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>380</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;
color : grey;</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Sportify-QT-project-master/add.png</normaloff>../../Sportify-QT-project-master/add.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;
color : grey;</string>
     </property>
     <property name="text">
      <string>PDF List</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Sportify-QT-project-master/pdf.png</normaloff>../../Sportify-QT-project-master/pdf.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>370</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
        font-size:16px;</string>
     </property>
     <property name="text">
      <string>Mdp</string>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimeragent">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>350</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;
color : grey;</string>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Sportify-QT-project-master/delete.png</normaloff>../../Sportify-QT-project-master/delete.png</iconset>
     </property>
    </widget>
    <widget class="QTableView" name="tableagent">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>180</y>
       <width>311</width>
       <height>192</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;
color : grey;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
        font-size:16px;</string>
     </property>
     <property name="text">
      <string>Prenom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>644</x>
       <y>30</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;
color : grey;</string>
     </property>
     <property name="text">
      <string>Gestion Partenaire</string>
     </property>
    </widget>
    <widget class="QPushButton" name="home_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;
background-color:#262525;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/Resources/back.png</normaloff>:/resources/Resources/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="styleSheet">
     <string notr="true">background-color:#121111;
</string>
    </property>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>410</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;
color : grey;</string>
     </property>
     <property name="text">
      <string> LISTE PARTENAIRE</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Sportify-QT-project-master/worker.png</normaloff>../../Sportify-QT-project-master/worker.png</iconset>
     </property>
    </widget>
    <widget class="QLineEdit" name="recherche_3">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>171</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : transparent;
border:none;
color : #717072;
border-bottom : 1px solid  #717072;
font-size: 18px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rechercher_3">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>170</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;
color : grey;</string>
     </property>
     <property name="text">
      <string>Rechercher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>634</x>
       <y>410</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;
color : grey;</string>
     </property>
     <property name="text">
      <string>load data</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Sportify-QT-project-master/load.png</normaloff>../../Sportify-QT-project-master/load.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
        font-size:16px;</string>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cin_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>340</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : transparent;
border:none;
color : #717072;
border-bottom : 1px solid  #717072;
font-size: 18px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>230</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : transparent;
border:none;
color : #717072;
border-bottom : 1px solid  #717072;
font-size: 18px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_mdp_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>390</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : transparent;
border:none;
color : #717072;
border-bottom : 1px solid  #717072;
font-size: 18px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>340</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
        font-size:16px;</string>
     </property>
     <property name="text">
      <string>CIN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>340</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;
color : grey;</string>
     </property>
     <property name="text">
      <string>PDF List</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Sportify-QT-project-master/pdf.png</normaloff>../../Sportify-QT-project-master/pdf.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter_Ag_3">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;
color : grey;</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Sportify-QT-project-master/add.png</normaloff>../../Sportify-QT-project-master/add.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>390</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
        font-size:16px;</string>
     </property>
     <property name="text">
      <string>Somme des dons</string>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimeragent_3">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>370</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;
color : grey;</string>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Sportify-QT-project-master/delete.png</normaloff>../../Sportify-QT-project-master/delete.png</iconset>
     </property>
    </widget>
    <widget class="QTableView" name="tableagent_3">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>200</y>
       <width>311</width>
       <height>192</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;
color : grey;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prenom_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>280</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : transparent;
border:none;
color : #717072;
border-bottom : 1px solid  #717072;
font-size: 18px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>280</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
        font-size:16px;</string>
     </property>
     <property name="text">
      <string>Adresse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>664</x>
       <y>30</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;
color : grey;</string>
     </property>
     <property name="text">
      <string>Gestion Agent</string>
     </property>
    </widget>
    <widget class="QPushButton" name="home_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;
background-color:#262525;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/Resources/back.png</normaloff>:/resources/Resources/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
